libvast/vast/system/explorer.hpp

```diff
@@ -26,6 +26,7 @@ struct explorer_state {
   struct event_limits {
     uint64_t total;
     uint64_t per_result;
+    uint64_t initial_query;
```

Comment:
```suggestion
    uint64_t taste_size;
```

The issue I have with "initial query" is that it doesn't reflect a quantity.

Reply:
Our [style guide](https://github.com/tenzir/.github/blob/master/contributing.md#naming) recommends prefixing these kinda variables with `num_`:

> Prefix counter variables with `num_`.

Reply:
How did *that* get into the guide? The CONTRIBUTING.md that was removed from this repository didn't have that but the one that was added to the other repo had it from the beginning?

In this case the surrounding instance variable is called `limits`, which already indicates that the contained values are numeric.